src/Components/Sidebar/Sidebar.jsx

```diff
@@ -11,13 +11,13 @@ function Sidebar() {
         <NavLink className="link" to="/main-page">
```

Comment:
Good that you use `NavLink` but I can't see any property attached to it, if you decide to use it in the future then it's all right👍

Reply:
Yes